rplidar.go

```diff
@@ -54,7 +54,11 @@ func init() {
 			if devicePath == "" {
 				return nil, errors.New("need to specify a devicePath (ex. /dev/ttyUSB0")
 			}
-			return NewRPLidar(logger, port, devicePath)
+			dataFolder := config.Attributes.String("data_folder")
+			if dataFolder == "" {
+				return nil, errors.New("need to specify a folder for the lidar data")
```

Comment:
I would mention the exact key name that's missing in the error message: "need to set data_folder to a valid storage location" or something like that.